VideoWeb/VideoWeb/ClientApp/src/app/services/conference/models/virtual-meeting-room.model.ts

```diff
@@ -0,0 +1,25 @@
+import { ParticipantModel } from 'src/app/shared/models/participant';
+import { RoomSummaryResponse } from '../../clients/api-client';
+import { PexipDisplayNameModel } from './pexip-display-name.model';
+
+export class VirtualMeetingRoomModel {
+    public pexipDisplayName: PexipDisplayNameModel | null;
+    constructor(
+        public id: string,
+        public displayName: string,
+        public locked: boolean,
+        public participants: ParticipantModel[] = [],
+        public pexipId: string | null = null,
+        pexipDisplayName: string | PexipDisplayNameModel | null = null
```

Comment:
should this not be consistent with all the other properties here and be declared in the constructor?

Reply:
No because two types can be passed in so there is some logic in the constructor. This lets the model be created with the pexip display name model or the string equivilent.